assets/app/view/game/history_controls.rb

```diff
@@ -46,6 +47,7 @@ def link(text, title, action_id = nil)
         route = Lib::Params.add(@app_route, 'action', action_id)
 
         click = lambda do
+          store(:tile_selector, nil)
```

Comment:
hm this is not the best, i wonder if there's any other way around it

Reply:
Another solution would be to store the action id in the selector and ignoring if it doesn't match.

Reply:
Alternatively move the tile selector and selected_company into a 'clear_ui_state' method in actionable